README.md

```diff
@@ -1 +1,36 @@
+
+## What does the name mean?
+
+"Choroscope" is a synthetic word derived from the Greek χώρα (chora), "place," and σκoπέω (skopeo) "to look at, contemplate, or examine." Choroscope is about studying data as distributed over geography.
+
+## How can I use Choroscope to visualize my own geospatial data set?
+
+You can't just yet, but as the platform is published, we plan to provide reference documentation, tutorials, and demos to help you set it up to serve your own needs.
+
+## What are the components of the Choroscope platform, and what is the plan for publishing them?
+
+The plan is to publish the individual components of Choroscope in sequence, proceding conceptually from bottom to top:
+
+- [`choroscope-spec`](https://github.com/choroscope/choroscope-spec) (available now) defines the configuration language for the platform. Every data "theme" in Choroscope needs to provide a configuration file to describe the data and how it should be visualized.
+- `choroscope-db` (coming soon) is a Python-based utility that automates the creation of a geospatial ([PostGIS](http://postgis.net/)) database representing a data "theme." Though this database is intended primarily to serve as the data source for the frontend application, it may also be queried directly using PostGIS SQL, providing a powerful and open-ended way of exploring a geospatial data set.
+- `choroscope-api` is a NodeJS server that makes the data contained in a Choroscope database (as created by `choroscope-db`) available as a web API. Though mainly intended to serve the needs of the frontend application, the API could also be used by other custom applications.
+- `choroscope-app` is a frontend web application that visualizes the data provided by `choroscope-api`. Written in [TypeScript](http://www.typescriptlang.org/), the application leverages the mapping framework [Leaflet](https://leafletjs.com/) as well as a variety of other open-source technologies. It also provides a custom [WebGL-based renderer](https://github.com/ihmeuw/leaflet.tilelayer.glcolorscale) for visualizing pixel data.
+
+For the Local Burden of Disease project, we've created additional resources for deploying the Choroscope application and its databases as containerized microservices. Our intent is that the platform itself be agnostic as to how and where it's deployed, but to simplify the setup process for people who want to create their own instances of Choroscope, we also plan to publish generalized deployment resources, probably in the form of a [Helm Chart](https://helm.sh/) for deployment in [Kubernetes](https://kubernetes.io/).
```

Comment:
This sentence runs on a bit: "Our intent is that the platform itself be agnostic as to how and where it's deployed, but to simplify the setup process for people who want to create their own instances of Choroscope, we also plan to publish generalized deployment resources, probably in the form of a Helm Chart for deployment in Kubernetes."

Maybe reword as follows:
"Our intent is that the platform itself be agnostic as to how and where it's deployed. In the future we plan to simplify the Choroscope setup process by publishing generalized deployment resources to assist with instantiation of the services, probably in the form of a Helm Chart for deployment in Kubernetes."

Reply:
Done, though I changed the wording a bit.